Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -562,6 +569,57 @@ private static void RangeMaxChangedCallback(DependencyObject d, DependencyProper
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the control is optimized for touch use.
+        /// </summary>
+        /// <value>
+        /// The value for touch optimization.
+        /// </value>
+        public bool IsTouchOptimized
+        {
+            get
+            {
+                return (bool)GetValue(IsTouchOptimizedProperty);
+            }
+
+            set
+            {
+                SetValue(IsTouchOptimizedProperty, value);
+            }
+        }
+
+        private static void IsTouchOptimizedChangedCallback(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var rangeSelector = d as RangeSelector;
+            if (rangeSelector == null)
+            {
+                return;
+            }
+
+            rangeSelector.ArrangeForTouch();
+        }
+
+        private void ArrangeForTouch()
+        {
+            if (_containerCanvas == null)
+            {
+                return;
+            }
+
+            if (IsTouchOptimized)
+            {
+                _outOfRangeContentContainer.Height = _minThumb.Height = _maxThumb.Height = 44;
+                _minThumb.Width = _maxThumb.Width = 44;
+                _minThumb.Margin = _maxThumb.Margin = new Thickness(-20, 0, 0, 0);
+            }
+            else
+            {
+                _outOfRangeContentContainer.Height = _minThumb.Height = _maxThumb.Height = 24;
+                _minThumb.Width = _maxThumb.Width = 8;
+                _minThumb.Margin = _maxThumb.Margin = new Thickness(-8, 0, 0, 0);
```

Comment:
I could not think of a more elegant solution here, there is no relation between the two cases (44-> 24, 20->8), any thoughts?